src/components/Search.jsx

```diff
@@ -0,0 +1,17 @@
+
+
+function Search() {
```

Comment:
Where are the props?

How are you going to handle search?

Reply:
1 import React from 'react'

2 function Search(props) {}

is it good?

Reply:
I cannot see the difference. If you not use props inside of the component it has no value.